centreon/www/include/monitoring/status/Services/xml/serviceXML.php

```diff
@@ -556,7 +556,7 @@ function analyseGraphs(array $hostServiceIds): void
         $obj->XML->writeElement("sc", $obj->colorService[$data["state"]]);
         $obj->XML->writeElement("cs", _($obj->statusService[$data["state"]]), false);
         $obj->XML->writeElement("ssc", $data["state"]);
-        $obj->XML->writeElement("po", htmlspecialchars(htmlspecialchars($pluginShortOuput)));
+        $obj->XML->writeElement("po", CentreonUtils::escapeSecure($pluginShortOuput));
```

Comment:
`escapeSecure` seems to only remove basic tags by default.
I don't see how it can work in xml.